docs/installation.md

```diff
@@ -36,10 +36,19 @@ To run it as a standalone process you only need to run the binary file downloade
 
 1. Run installation script:
 
+    Either by piping `curl` straight into `bash`:
+
     ```bash
-    curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh | sudo bash -s -- --installation-token "${SUMOLOGIC_INSTALL_TOKEN}"
+    curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh | sudo --preserve-env=SUMOLOGIC_INSTALL_TOKEN bash
```

Comment:
Should we just do `-E` instead to take the whole environment?

Reply:
I wanted to explicitly only transfer this env var, as this is the only one used by the installation script.

Reply:
I'm worried that we'll forget to update this command after adding more.

Reply:
The form `--preserve-env=SUMOLOGIC_INSTALL_TOKEN` makes it explicit why we're using this argument. But I can change it to just `-E` and just explain in the document why it's needed.

Reply:
Done